sources/api/host-containers/src/main.rs

```diff
@@ -434,22 +434,47 @@ where
     Ok(())
 }
 
+fn is_restart_required(changed_settings: &String, container_name: &str) -> bool {
```

Comment:
nit: Just a preference and better consistency with the rest of the code-base. 
`&String` can be coerced into `&str` but not the other way around I think?
```suggestion
fn is_restart_required(changed_settings: &str, container_name: &str) -> bool {
```